### DISCUSSION TITLE
Just a Question for nested JSON

### DISCUSSION THREAD

**Discussion by NicosNicolaou16** [square/retrofit#4132] [Q&A] [answered]
Hello,
Is there a way to serialize a three times nested JSON as a flat, I mean to get the specific value directly to avoid create nested models?
I searched about it but I didn't find anything clear!
Thank you!

**Comment by JakeWharton** [comment, marked as answer]
Retrofit is not involved in the deserialization of JSON, and so you'll have to go ask on the support forums for your chosen JSON library. All we do is hand the raw body bytes to your chosen converter and wait for it to give us back the resulting object. Any customization of how the bytes are turned into the object is entirely done within that library, not ours. Good luck!

**Reply by NicosNicolaou16** [reply]
Thank you for your response 🙏😃
